### PR TITLE
Finished 4 isolated consensus nodes

### PR DESCRIPTION
**Pull request by nunojusto**
NOTE: It can only be merged after this PR https://github.com/CityOfZion/neo-local/pull/90

This PR is to merge all the work for devlop and delete the 4-isolated-nodes branch

### CODE REVIEW

**Comment by nunojusto** [comment]
@revett after some headache trying to git sync back and forth i think it's all done.

**Comment by nunojusto** [comment]
This needs #90 first and then @revett to have some time to approve it

File: docker-compose.yml [hunk 1]
```diff
@@ -13,23 +13,65 @@ services:
     ports:
       - "4002:4002"
 
-  neo-nodes:
-    container_name: neo-nodes
-    image: cityofzion/neo-privatenet:2.7.6
+  neo-privatenet1:
+    container_name: neo-privatenet1
+    image: cityofzion/neo-local-privatenet:$NODE1_NEO_CLI_VERSION
+    #build:
```
Comment (revett) [inline docker-compose.yml]: Remove comments

File: docker-compose.yml [hunk 2]
```diff
@@ -13,23 +13,65 @@ services:
...
       - "20333:20333"
-      - "20334:20334"
-      - "20335:20335"
-      - "20336:20336"
       - "30333:30333"
+
+  neo-privatenet2:
+    container_name: neo-privatenet2
+    image: cityofzion/neo-local-privatenet:$NODE2_NEO_CLI_VERSION
+    #build:
```
Comment (revett) [inline docker-compose.yml]: Remove comments

File: docker-compose.yml [hunk 3]
```diff
@@ -13,23 +13,65 @@ services:
...
+    environment:
+      NODE_NUMBER: 2
+    ports:
+      - "20334:20334"
       - "30334:30334"
+
+  neo-privatenet3:
+    container_name: neo-privatenet3
+    image: cityofzion/neo-local-privatenet:$NODE3_NEO_CLI_VERSION
+    #build:
```
Comment (revett) [inline docker-compose.yml]: Remove comments

File: docker-compose.yml [hunk 4]
```diff
@@ -13,23 +13,65 @@ services:
...
+    environment:
+      NODE_NUMBER: 3
+    ports:
+      - "20335:20335"
       - "30335:30335"
+  
+  neo-privatenet4:
+    container_name: neo-privatenet4
+    image: cityofzion/neo-local-privatenet:$NODE4_NEO_CLI_VERSION
+    #build:
```
Comment (revett) [inline docker-compose.yml]: Remove comments.

File: docker-compose.yml [hunk 5]
```diff
@@ -13,23 +13,65 @@ services:
     ports:
       - "4002:4002"
 
-  neo-nodes:
-    container_name: neo-nodes
-    image: cityofzion/neo-privatenet:2.7.6
+  neo-privatenet1:
```
Comment (revett) [inline docker-compose.yml]: Change naming of all to:

```
neo-privatenet-x
```

**Comment by nunojusto** [comment]
https://github.com/CityOfZion/neo-local/issues/47

**Pull request merged**